utils/products.js

```diff
@@ -1,152 +1,372 @@
+    productDescription: "Tractus-X Resolver Service for BPN <> DID resolution",
+    githubRepo: [
+      "https://github.com/eclipse-tractusx/bpn-did-resolution-service",
+    ],
+    committers: [
+      "https://github.com/paullatzelsperger",
+      "https://github.com/wolf4ood",
+      "https://github.com/jimmarino",
+    ],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request bpn-did-resolution-service Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "demand-capacity-mgmt",
+    productDescription: "TBD",
+    githubRepo: ["https://github.com/eclipse-tractusx/demand-capacity-mgmt"],
+    committers: ["https://github.com/nitin-vavdiya"],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request demand-capacity-mgmt Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "Digital Product Pass",
+    productDescription: 
+    "The Digital Product Pass (DPP) Application provides an easy way to request and visualize product passports from an economic operator using the Catena-X Network. By scanning a QR code or introducing a manufacturerPartId and partInstanceId (productId) passports can be displayed for different products like Batteries (Battery Pass), Transmissions (Transmision Pass) and any other product by simply using the Generic Digital Product Passport Model. Additionally components like the dpp-backend have the power to retrieve any aspect submodel content, which is registered in a Digital Twin using the Catena-X Network, and the dpp-verification add-on enables certification + verification of Catena-X Instance/Type Aspect Models",
+    githubRepo: ["https://github.com/eclipse-tractusx/digital-product-pass"],
+    committers: [
+      "https://github.com/matbmoser",
+      "https://github.com/saudkhan116",
     ],
-    mailTo: "tractusx-dev@eclipse.org?subject=Request Business Partner Data Management Team",
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request digital-product-pass Team",
+    hasBoard: true,
+    showVersion: true,
+  },
+  {
+    productName: "identity-trust",
+    productDescription: "TBD",
+    githubRepo: ["https://github.com/eclipse-tractusx/identity-trust"],
+    committers: [
+      "https://github.com/jimmarino",
+      "https://github.com/wolf4ood",
+      "https://github.com/paullatzelsperger",
+    ],
+    mailTo: "tractusx-dev@eclipse.org?subject=Request identity-trust Team",
     hasBoard: false,
-    showVersion: true
+    showVersion: true,
   },
-
   {
     productName: "IDS Essential Services",
-    productDescription: "The project provides additional services to enable an IDSA-infrastructure within the Catena-X dataspace.",
+    productDescription:
+      "The project provides additional services to enable an IDSA-infrastructure within the Catena-X dataspace.",
     githubRepo: [
       "https://github.com/eclipse-tractusx/daps-registration-service",
       "https://github.com/eclipse-tractusx/daps-helm-chart",
```

Comment:
the two repos should be removed and the product be changed to Self-Description Factory.